201-file-share-private-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,1073 @@
+      "location": "global",
+      "dependsOn": [
+        "[variables('blobPrivateDnsZoneId')]",
+        "[variables('vnetId')]"
+      ],
+      "properties": {
+        "registrationEnabled": false,
+        "virtualNetwork": {
+          "id": "[variables('vnetId')]"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Network/privateEndpoints",
+      "apiVersion": "2020-07-01",
+      "name": "[parameters('blobStorageAccountPrivateEndpointName')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[variables('vnetId')]",
+        "[variables('blobStorageAccountId')]"
+      ],
+      "properties": {
+        "privateLinkServiceConnections": [
+          {
+            "name": "[parameters('blobStorageAccountPrivateEndpointName')]",
+            "properties": {
+              "privateLinkServiceId": "[variables('blobStorageAccountId')]",
+              "groupIds": [
+                "[variables('blobStorageAccountPrivateEndpointGroupName')]"
+              ]
+            }
+          }
+        ],
+        "subnet": {
+          "id": "[variables('subnetId')]"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups",
+      "apiVersion": "2020-07-01",
+      "name": "[variables('blobPrivateDnsZoneGroup')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[variables('blobPrivateDnsZoneId')]",
+        "[variables('blobStorageAccountPrivateEndpointId')]"
+      ],
+      "properties": {
+        "privateDnsZoneConfigs": [
+          {
+            "name": "dnsConfig",
+            "properties": {
+              "privateDnsZoneId": "[variables('blobPrivateDnsZoneId')]"
+            }
+          }
+        ]
+      }
+    }
+  ],
+  "outputs": {
+    "storageAccountPrivateEndpoint": {
+      "value": "[reference(resourceId('Microsoft.Network/privateEndpoints', parameters('fileStorageAccountPrivateEndpointName')), '2020-07-01', 'Full')]",
+      "type": "object"
+    },
+    "blobStorageAccountPrivateEndpoint": {
+      "value": "[reference(resourceId('Microsoft.Network/privateEndpoints', parameters('blobStorageAccountPrivateEndpointName')), '2020-07-01', 'Full')]",
+      "type": "object"
+    },
+    "fileStorageAccount": {
+      "value": "[reference(resourceId('Microsoft.Storage/storageAccounts', parameters('fileStorageAccountName')), '2019-06-01', 'Full')]",
+      "type": "object"
+    },
+    "blobStorageAccount": {
+      "value": "[reference(resourceId('Microsoft.Storage/storageAccounts', parameters('blobStorageAccountName')), '2019-06-01', 'Full')]",
+      "type": "object"
+    },
+    "adminUsername": {
+      "value": "[parameters('adminUsername')]",
+      "type": "string"
+    },
+    "workspaceName": {
+      "value": "[parameters('workspaceName')]",
+      "type": "string"
+    },
+    "scriptFileUri": {
+      "value": "[variables('scriptFileUri')]",
+      "type": "string"
+    },
+    "environment": {

```

Comment:
remove this from outputs

Reply:
Fixed. Can you eventually be more specific about the outputs I can keep? GitHub UI doesn't help much to understand which elements you are asking me to remove. I removed scriptFileUri and environment.

Reply:
I removed all the outputs except the **adminUsername** and **adminPasswordOrKey**. The best practice guide does provide any guidance about allowed and disallowed outputs. Maybe it would be good to add some guidelines to the [best practices](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)